### ISSUE TITLE
Prevent errors produced by tool if invalid date is selected

### ISSUE THREAD

**Issue by bstaton1** [bstaton1/KuskoHarvPred#16]
The date selector uses June 12 to July 31 as the range allowed. If a date outside this is selected, the outputs tab looks like this:

![image](https://github.com/bstaton1/KuskoHarvPred/assets/42722538/bfe57a28-dc29-4ef1-8791-5d040dbb747c)

This should be updated to include a nice flag instructing the user to enter a valid date.

**Comment by bstaton1** [comment]
I solved this by forcing the `default_date` to be between the `min_date` and `max_date`:

```R
if (default_date > max_date) default_date = max_date
if (default_date < min_date) default_date = min_date
```

I found this to be **much** simpler than trying to conditionally control what happens if an invalid date is used. This "edge" case only comes up when the tool is used outside of the window of June 12 and July 31 as it is.

**Comment by bstaton1** [comment]
Closed in #17
